packages/web/src/pages/sign-up-page/components/NavHeader.tsx

```diff
@@ -93,23 +95,44 @@ export const NavHeader = () => {
               variant={PlainButtonType.SUBDUED}
             />
           ) : null}
-        </Route>
-        {!isMobile ? (
-          <Route
-            path={[
-              SIGN_UP_HANDLE_PAGE,
-              SIGN_UP_FINISH_PROFILE_PAGE,
-              SIGN_UP_GENRES_PAGE,
-              SIGN_UP_ARTISTS_PAGE
-            ]}
-          >
-            {audiusLogo}
+        </HeaderRoot>
+      </Route>
+      {!isMobile ? (
+        <Route
+          path={[
+            SIGN_UP_HANDLE_PAGE,
+            SIGN_UP_FINISH_PROFILE_PAGE,
+            SIGN_UP_GENRES_PAGE,
+            SIGN_UP_ARTISTS_PAGE
+          ]}
+        >
+          <HeaderRoot justifyContent='center' pv={undefined} pt='l'>
```

Comment:
`pv={undefined}` feels weird. Should we be accepting null or `0` in the props?

Reply:
if this is just a one-off, maybe use css prop instead?

Reply:
ohh i like that cause it's a one-off, its basically overriding the default pv that is defined in headerRoot. ideally we should be able to pass null so we can "cancel" it. what do you think? that or just css override?

Reply:
I think allowing HeaderRoot to take a value that overrides the default is good. Passing undefined just seemed odd when we really mean zero